webplugin/js/app/zendesk-chat-service.js

```diff
@@ -74,6 +74,25 @@ function ZendeskChatService() {
                     _this.handleZendeskAgentMessageEvent(eventDetails);
                 } else if (eventDetails.type == "chat.file") { //If agent sends file attachments
                     _this.handleZendeskAgentFileSendEvent(eventDetails);
+                } else if (eventDetails.type == "chat.memberleave") {
+                    //Resolve conversation on widget
+                    KommunicateConstants.CONVERSATION_RESOLVED_STATUS = true;
```

Comment:
i guess this is a constant. are we supposed to change constants as well?

Reply:
No it wasn't necessary. Removed it.